pkg/reconciler/trigger/resources/scaled_object.go

```diff
@@ -15,6 +15,31 @@ type DispatcherScaledObjectArgs struct {
 	Trigger              *v1beta1.Trigger
 }
 
+type RabbitTriggerAuthenticationArgs struct {
+	Namespace string
+}
+
+const TriggerAuthenticationName = "rabbit-trigger-authentication"
+
+func MakeRabbitTriggerAuthentication(args *RabbitTriggerAuthenticationArgs) *kedav1alpha1.TriggerAuthentication {
+	return &kedav1alpha1.TriggerAuthentication{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: args.Namespace,
+			Name:      TriggerAuthenticationName,
```

Comment:
I was just thinking that this should have the Broker as its OwnerReference (and so should the ScaleTriggerSecret as that is also going to be re-used by any Trigger on that Broker).

Then of course that raises the point that those 2 resources should probably be created under the "broker" umbrella (its `reconciler.go` and its `resources` sub-package).

WDYT @fbiville ?

Reply:
it makes sense indeed!